### PR TITLE
Add Exercise02

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise02/Program.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+namespace Exercise02
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var data = new CreateData();
+
+            var listOfStudent = data.GetStudents();
```
Comment (voquanghoa) [inline Exercise02/Program.cs]: Use plural form

File: Exercise02/Program.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            listOfStudent.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var listOfClass = data.GetClasses();
```
Comment (voquanghoa) [inline Exercise02/Program.cs]: Use plural form

File: Exercise02/Program.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            listOfClass.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var list = listOfStudent.OrderBy(x=>x.ClassId).ThenBy(x=>x.Name).ToList();
```
Comment (voquanghoa) [inline Exercise02/Program.cs]: Bad name

File: Exercise02/Program.cs [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            list.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var listHasStudentCount = listOfClass.GroupJoin(
```
Comment (voquanghoa) [inline Exercise02/Program.cs]: Try to use [linq syntax](https://www.dotnetperls.com/join)

File: Exercise02/Program.cs [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+                                    Name = x.Name
+                                }).ToList();
+            listHasAverageScore.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var listOfTop5Student = listOfStudent.OrderByDescending(x=>x.Score).Take(5).ToList();
+            listOfTop5Student.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var listOfTop5StudentHasSroceGreater6 = listOfStudent.Where(x => x.Score >= 6).OrderByDescending(x => x.Score).Take(3).ToList();
```
Comment (voquanghoa) [inline Exercise02/Program.cs]: Enter a new line when you call a function
```
var xxx = listOfStudent
      .Where(x => x.Score >= 6)
      .OrderByDescending(x => x.Score)
      .Take(3)
      .ToList();
```

File: Exercise02/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise02/Program.cs]: Done

File: Exercise02/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise02/Program.cs]: Done

File: Exercise02/Program.cs (hunk 4 above)
Comment (huynhngocminh2511) [reply Exercise02/Program.cs]: Done

File: Exercise02/Program.cs (hunk 5 above)
Comment (huynhngocminh2511) [reply Exercise02/Program.cs]: Done

File: Exercise02/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise02/Program.cs]: Done

**Pull request merged**